### ISSUE TITLE
g8 scalatra/scalatra-sbt failing

### ISSUE THREAD

**Issue by FranGoitia** [scalatra/scalatra-sbt.g8#66]
Trying to initiate a new Scalatra project in Mac OS X and sbt 0.13.13 and giter8 0.8.0. This happens:

```
LT-R13-0107:shapelets_api_2 franciscog$ g8 scalatra/scalatra-sbt
organization [com.example]: rga
package [com.example.app]: shapelets_api
name [My Scalatra Web App]: Shapelets API
scalatra_version [2.5.0]: 2.5.0
servlet_name [MyScalatraServlet]: ShapeletsApi
scala_version [2.12.1]: 2.12.1
version [0.1.0-SNAPSHOT]: 0.1.0-SNAPSHOT
sbt_version [0.13.13]: 0.13.13
(jar_file \ []: shapelets_api
{SBT_OPTS:-\ []:
(make_url \ []:
jvm_opts_file" ]] && cat "\ []:
category/\ []:
(( \ []:
{perm}m -XX:ReservedCodeCacheSize=\ []:
sbt_jar" ]] || download_url "\ []:
{sbt_commands[@]}" "\ []:
SCALATRA_JREBEL -Dscalatra_project_root=\ []:
jar" "\ []:
{scalac_args[@]}" "\ []:
path" ]] || { echo "\ []:
debug ]] && echoerr "\ []:
{#scalac_args[@]} -eq 0 ]] || addSbt "set scalacOptions in ThisBuild += \"\ []:
(get_mem_opts \ []:
arg" ]] || [[ "\ []:
url" --output "\ []:
{residual_args[@]}" "\ []:
sbt_launch_dir/\ []:
1" "\ []:
(readlink "\ []:
{java_args[@]}" "\ []:
(dirname \ []:
argumentCount > 0 )) || echo "Starting \ []:
scala_version" ]] && echo "Overriding scala version to \ []:
verbose || \ []:
{sbtargs[@]}" "\ []:
(sbt_groupid) releases \ []:
(dirname "\ []:
{JAVA_OPTS:-\ []:
quiet ]] && echo 1>&2 "\ []:
( cd "\ []:
(basename \ []:
/sbt.version=\ []:
opt requires <\ []:
ver == \ []:
 cd  []:
{mem}m -Xmx\ []:
sbt_explicit_version" ]] && update_build_props_sbt "\ []:
(get_script_path "\ []:
cannot have '.' in attribute names
```

**Comment by jwnorma** [comment]
I just ran into the same issue, but was able to get the template to work using sbt's new built in `new` command:

`sbt new scalatra/scalatra-sbt.g8`

worked w/ latest sbt `0.13.15`

**Comment by takezoe** [comment]
@FranGoitia has raised same issue at https://github.com/scalatra/scalatra/issues/650 and already it was solved. Could you close this issue as well?

**Comment by mihbor** [comment]
same issue here with g8, with "sbt new" it works.
I suggest updating the documentation before closing this issue.

**Comment by aadrian** [comment]
@mihbor the pull requests for this https://github.com/scalatra/scalatra-sbt.g8/pulls are a month old there.

**Comment by mihbor** [comment]
:)

**Comment by xuwei-k** [comment]
I think fixed in giter8 0.9.0

**Comment by takezoe** [comment]
Yep. Also document have been updated to use `sbt new`.
